test/verify/check-openshift

```diff
@@ -904,5 +904,69 @@ class TestRegistry(MachineCase):
         # Make sure it showed up in the console
         wait_project(o, "llama");
 
+    def testDockerCommandInfo(self):
+        o = self.openshift
+        b = self.browser
+        m = self.machine
+
+        # create push and pull user and login as pushuser
+        o.execute("oadm policy add-role-to-user registry-viewer pulluser -n marmalade")
+        o.execute("oadm policy add-role-to-user registry-editor pushuser -n marmalade")
+        tmpfile = os.path.join(self.tmpdir, "kubeconfig")
+        o.execute('printf "pushuser\r\na\r\n" | oc login')
+        o.download("/root/.kube/config", tmpfile)
+        with open(tmpfile, "r") as f:
+            m.execute("mkdir -p /home/admin/.kube && cat > /home/admin/.kube/config", input=f.read())
+
+        self.login_and_go("/kubernetes/registry")
+
+        # always visible on "All projects" page
+        b.wait_in_text("body", "Pull an image")
+        b.wait_visible('#docker-push-commands')
+        b.wait_visible('#docker-pull-commands')
+
+        # push user should see docker push and pull commands on project specific  overview page
+        b.click(".dashboard-images .namespace-filter button")
+        b.wait_visible(".dashboard-images .namespace-filter .dropdown-menu")
+        b.wait_present(".dashboard-images .namespace-filter a[value='marmalade']")
+        b.click(".dashboard-images .namespace-filter a[value='marmalade']")
+        time.sleep(2)  # FIXME: better way to say "assert that this does not disappear"?
```

Comment:
How about first selecting a project that the user does not have access to, checking it goes away and then making sure selecting this one brings it back?